pgxn/neon/neon.control

```diff
@@ -1,6 +1,6 @@
 # neon extension
 comment = 'cloud storage for PostgreSQL'
-default_version = '1.5'
+default_version = '1.6'
```

Comment:
Please, add the PR description explaining why we think it's fine to bump the default

Reply:
Fixed